go/libkb/globals.go

```diff
@@ -176,14 +176,21 @@ func (g *GlobalContext) ConfigureKeyring() error {
 	return nil
 }
 
-func VersionMessage(linefn func(string)) {
+type GitHashGetter interface {
+	Hash() string
+}
+
+func (g *GlobalContext) VersionMessage(linefn func(string)) {
 	linefn(fmt.Sprintf("Keybase CLI %s", VersionString()))
 	linefn(fmt.Sprintf("- Built with %s", runtime.Version()))
+	if h, ok := interface{}(g).(GitHashGetter); ok {
```

Comment:
I'm attempting to ask the GlobalContext whether it implements GitHashGetter, in a way that would be OK if it does not. It seemed I had to first copy it as an empty interface because it is declared as a struct, not an interface. Maybe there is a better way? I saw something similar on the internet.